Mondee Watch App/View/Play/GameOptionView.swift

```diff
@@ -39,10 +52,47 @@ struct GameOptionView: View {
                 }
                 .frame(maxHeight: .infinity, alignment: .center)
                 .edgesIgnoringSafeArea(.bottom)
+                .alert("게임을 포기하시겠어요?", isPresented: $gameTerminationAlert) {
+                    Button(role: .destructive) {
+                        gameState.giveUpGame()
+                        gameStatus = .fail
+                        withAnimation() {
+                            selection = .game
+                        }
+                    } label: {
+                        Text("오늘 그만")
+                    }
+
+                    Button(role: .cancel) {
+                        
+                    } label: {
+                        Text("취소")
+                    }
```

Comment:
action 버튼의 기능이 없다면 이렇게 줄여보면 어떨까요?
```suggestion
                   Button("취소", role: .cancel) {}    
```